docs/manual-CN/3.build-develop-and-administration/1.build/1.build-source-code.md

```diff
@@ -91,7 +92,7 @@ bash> make
     ```
     **现在可以开始运行 Nebula** 。
 
-### 使用 docker 容器构建
+<!-- ### 使用 docker 容器构建
 
```

Comment:
why comment this out ?

Reply:
this doc only cares `build from source code`.

Reply:
ok